src/lib.rs

```diff
@@ -67,7 +67,8 @@
     specialization,
     trusted_len,
     unsized_locals,
-    fn_traits
+    fn_traits,
+    exhaustive_patterns
```

Comment:
As far as I can see, there is no rustc version, which supports clippy and rustfmt with the never_type stabilized. We might want to add `never_type` as feature until this is resolved.
```suggestion
    exhaustive_patterns,
    never_type
```

Reply:
Clippy works with todays nightly